packages/evm-adapter/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@heliofi/evm-adapter",
+  "version": "0.0.3-beta",
+  "description": "API to interact with Helio Finances program on EVM chains.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "yarn build -w",
+    "lint": "eslint .",
+    "lint:fix": "yarn lint --fix",
+    "test": "yarn run ts-mocha -t 1000000 test/*.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/heliofi/solidity-adapter.git"
+  },
+  "keywords": [
+    "ethereum",
+    "polygon",
+    "crypto",
+    "stream",
+    "payments"
+  ],
+  "license": "AGPL-3.0-or-later",
+  "bugs": {
+    "url": "https://github.com/heliofi/solidity-adapter/issues"
+  },
+  "homepage": "https://github.com/heliofi/solidity-adapter",
+  "dependencies": {
+    "ethers": "^5.7.2"
+  },
+  "author": "Ivan Britvic<ivbritvic@gmail.com>",
+  "devDependencies": {
+    "@ethersproject/abi": "^5.4.7",
+    "@ethersproject/providers": "^5.4.7",
+    "@heliofi/eslint-config-helio": "^0.0.4",
```

Comment:
this is a very old version can you please use the new one

Reply:
Hm, you're right don't know why hardhat initialized old version, moving ethers to 5.7.0